byron/chain/executable-spec/src/Cardano/Spec/Chain/STS/Rule/SigCnt.hs

```diff
@@ -32,6 +32,7 @@ instance STS SIGCNT where
 
   type Signal SIGCNT = VKey
 
+  -- | These `PredicateFailure`s are all throwable.
   data PredicateFailure SIGCNT
     = TooManyIssuedBlocks VKeyGenesis -- The given genesis key issued too many blocks.
```

Comment:
Since you're at it, maybe we can fix the haddocks for `TooManyIssuedBlocks` (missing `^`).

Reply:
Agh! Stupid sexy green button! I didn't want to start a review :/

Reply:
Fixed 🙂